src/renderer/desktop-capturer.ts

```diff
@@ -120,7 +120,9 @@ export const getSource = async (options: ICustomSourcesOptions, callback: Callba
 
     }
 
-    const updatedSources = sources.map((source) => {
+    const updatedSources = sources
+        .filter((source) => isWindowsOS ? source.name !== NOTIFICATION_WINDOW_TITLE : true)
```

Comment:
we use custom notifications on macOS as well now right? Should you remove the `isWindowsOS` condition here?

Reply:
@VishwasShashidhar MacOS is not considering `notification windows` as applications. so, the issue is only happening on Windows